apps/system/js/notifications.js

```diff
@@ -380,37 +380,41 @@ var NotificationScreen = {
   },
 
   addNotification: function ns_addNotification(detail) {
-    // LockScreen window may not opened while this singleton got initialized.
-    this.lockScreenContainer = this.getLockScreenContainer();
+  // LockScreen window may not opened while this singleton got initialized.
```

Comment:
This 'change' violates Mozilla coding style requirements. And it seems not required for feature implementation, since only formatting was changed. You shall avoid unnecessary changes in your commits, since it makes review of actual changes not trivial.